modules/openapi-generator/src/main/resources/kotlin-client/libraries/multiplatform/build.gradle.kts.mustache

```diff
@@ -1,17 +1,17 @@
 import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
 
 plugins {
-    kotlin("multiplatform") version "1.5.10" // kotlin_version
-    kotlin("plugin.serialization") version "1.5.10" // kotlin_version
+    kotlin("multiplatform") version "1.5.31" // kotlin_version
+    kotlin("plugin.serialization") version "1.5.31" // kotlin_version
 }
 
 group = "{{groupId}}"
 version = "{{artifactVersion}}"
 
-val kotlin_version = "1.5.10"
-val coroutines_version = "1.5.0"
-val serialization_version = "1.2.1"
-val ktor_version = "1.6.0"
+val kotlin_version = "1.5.31"
+val coroutines_version = "1.5.2"
+val serialization_version = "1.3.0-RC"
```

Comment:
@tkirshboim could you please update this to the stable version 1.3.1?

Reply:
@4brunu I can do that, but I chose this version because it is the [recommended](https://kotlinlang.org/docs/releases.html#release-details) version to use with Kotlin 1.5.31 .

Reply:
I think we should use here the stable version 1.3.0.

Reply:
✅ Done.